client/state/selectors/is-two-step-sms-enabled.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Is two-step sms enabled for the current user?
+ *
+ * @param  {Object} state Global state tree
+ * @return {Boolean} return true if two-step sms is enabled
+ */
+export default function isTwoStepSmsEnabled( state ) {
+	return get( state, 'userSettings.settings.two_step_sms_enabled', false );
```

Comment:
is this a safe default? seems like _if_ someone has two-step enabled but for some reason there's an issue getting that value out of Redux then disabling two-step seems like weak security. granted, I suppose that any valid API calls would fairytales but the UX could suffer. of course, it could suffer if we ask for two-step and it doesn't exist…

any thoughts?